ts/navigation/AuthenticationNavigator.tsx

```diff
@@ -98,6 +101,28 @@ const AuthenticationStackNavigator = () => (
       name={ROUTES.CIE_PIN_TEMP_LOCKED_SCREEN}
       component={CiePinLockedTemporarilyScreen}
     />
+
+    <Stack.Group
+      screenOptions={{
+        presentation: "modal",
+        headerLeft: () => null,
+        headerTitle: () => null,
+        headerRight: CloseButton,
+        headerStyle: {
+          height: IOVisualCostants.headerHeight,
```

Comment:
Can we remove this provided that we set `headerStatusBarHeight: 0`? What do you think?

```suggestion
```

Reply:
yes, correct! Thanks!